customization/customizing-the-api.md

```diff
@@ -128,18 +128,20 @@ Let's create our controller, along with its view, route a request spec to test i
 
 {% tabs %}
 {% tab title="product\_likes\_controller.rb" %}
-{% code title="app/controllers/api/product\_likes\_controller.rb" %}
+{% code title="app/controllers/api/products/product\_likes\_controller.rb" %}
```

Comment:
Why this change? I think it should match the module inside of it, so `app/controllers/api/product\_likes\_controller` was looking good to me. Any reason to change that I'm missing? Maybe they are both missing the `spree`/`Spree` folder/module?

Reply:
You are right, the name spacing should be spree/api/product_likes_controller.rb
Updated to reflect the change